### PR TITLE
Add line about journey dates in Swagger docs

### PR DESCRIPTION
**Pull request by thomasleese**
This explains what value it takes if not specified.

[Jira Ticket](https://dsdmoj.atlassian.net/browse/P4-3425)

### CODE REVIEW

File: spec/swagger/swagger_doc_v1.yaml [hunk 1]
```diff
@@ -1600,7 +1600,9 @@
             The journey object to be created. It must specify `timestamp`,
             `billable`, `from_location` and `to_location`. Optionally, `supplier`
             can also be specified if required (otherwise the supplier will be taken
-            from the logged-in account).
+            from the logged-in account). `date` can also be specified to represent
+            when the journey has or will take place (otherwisw this will be take
```
Comment (Thource) [inline spec/swagger/swagger_doc_v1.yaml:1604]: otherwise*

Comment (thomasleese) [reply spec/swagger/swagger_doc_v1.yaml]: Oh good catch!

File: spec/swagger/swagger_doc_v1.yaml [hunk 2]
```diff
@@ -1600,7 +1600,9 @@
             The journey object to be created. It must specify `timestamp`,
             `billable`, `from_location` and `to_location`. Optionally, `supplier`
             can also be specified if required (otherwise the supplier will be taken
-            from the logged-in account).
+            from the logged-in account). `date` can also be specified to represent
+            when the journey has or will take place (otherwise this will be take
```
Comment (Thource) [inline spec/swagger/swagger_doc_v1.yaml:1604]: Sorry, didn't see this first time, taken*?

Comment (thomasleese) [reply spec/swagger/swagger_doc_v1.yaml]: 🤦🏻

**Review by Thource — APPROVED** [review]

**Pull request merged**